src/Compilers/CSharp/Test/Semantic/Semantics/LambdaDiscardParametersTests.cs

```diff
@@ -351,6 +351,41 @@ public static void Main()
                 );
         }
 
+        [Fact]
+        public void DiscardParameters_WithAttribute()
+        {
+            var source =
+@"using System;
+class AAttribute : Attribute { }
+class C
+{
+    static void Main()
+    {
+        Action<object, object> a;
+        a = ([A] _, y) => { };
```

Comment:
Consider verifying that these attributes actually show up in the symbol model.

Reply:
Added verification here and in `LambdaTests.LambdaAttributes_01`.

---
In reply to: [605841651](https://github.com/dotnet/roslyn/pull/52178#discussion_r605841651) [](ancestors = 605841651)